apps/communications/contacts/style/matching_contacts.css

```diff
@@ -114,6 +113,16 @@ form[role="dialog"][data-type="edit"] #main {
   border-style: solid;
 }
 
+/* Ellipsis direction hack before bug 883884 is fixed */
+.ellipsis-dir-fix {
```

Comment:
I have the feeling that Steve already added this class, but to the main contacts, this is happening in a different document, and unfortunately we don't share much css there, so it's welcome :S